### PR TITLE
- setup

### PR DESCRIPTION
**Pull request by orlowski11**
Created initial setup for the application

### CODE REVIEW

**Comment by JakubKermes** [comment]
Maybe we could start with Laravel 11?

File: .env.example [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+PUSHER_PORT=443
+PUSHER_SCHEME=https
+PUSHER_APP_CLUSTER=mt1
+
+VITE_APP_NAME="${APP_NAME}"
+VITE_PUSHER_APP_KEY="${PUSHER_APP_KEY}"
+VITE_PUSHER_HOST="${PUSHER_HOST}"
+VITE_PUSHER_PORT="${PUSHER_PORT}"
+VITE_PUSHER_SCHEME="${PUSHER_SCHEME}"
+VITE_PUSHER_APP_CLUSTER="${PUSHER_APP_CLUSTER}"
```
Comment (kamilpiech97) [inline .env.example]: Pusher will be used in app?

File: database/seeders/DatabaseSeeder.php [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+namespace Database\Seeders;
+
+// use Illuminate\Database\Console\Seeds\WithoutModelEvents;
+use Illuminate\Database\Seeder;
+
+class DatabaseSeeder extends Seeder
+{
+    /**
+     * Seed the application's database.
+     */
```
Comment (kamilpiech97) [inline database/seeders/DatabaseSeeder.php]: Lets use [Blumilk codestyle](https://github.com/blumilksoftware/codestyle) for some cleaning in project

File: database/seeders/DatabaseSeeder.php [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    public function run(): void
+    {
+        // \App\Models\User::factory(10)->create();
+
+        // \App\Models\User::factory()->create([
+        //     'name' => 'Test User',
+        //     'email' => 'test@example.com',
+        // ]);
```
Comment (kamilpiech97) [inline database/seeders/DatabaseSeeder.php]: Is it need?

File: docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,55 @@
+services:
+    laravel.test:
+        build:
+            context: ./vendor/laravel/sail/runtimes/8.3
+            dockerfile: Dockerfile
+            args:
+                WWWGROUP: '${WWWGROUP}'
```
Comment (kamilpiech97) [inline docker-compose.yml]: Are u ok with sail?
Maybe lets use our [boilerplate](https://github.com/blumilksoftware/boilerplate).

File: composer.json [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+        "inertiajs/inertia-laravel": "^0.6.8",
+        "laravel/framework": "^10.10",
+        "laravel/sanctum": "^3.2",
+        "laravel/tinker": "^2.8",
+        "tightenco/ziggy": "^2.0"
+    },
+    "require-dev": {
+        "fakerphp/faker": "^1.9.1",
+        "laravel/breeze": "^1.29",
+        "laravel/pint": "^1.0",
```
Comment (kamilpiech97) [inline composer.json]: Lest use Blumilk codestyle - https://github.com/blumilksoftware/codestyle

File: docker-compose.yml [hunk 6]
```diff
@@ -0,0 +1,55 @@
+services:
+    laravel.test:
```
Comment (kamilpiech97) [inline docker-compose.yml]: The naming of services and other things seems to refer to the project name

File: app/Http/Requests/Auth/LoginRequest.php [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\RateLimiter;
+use Illuminate\Support\Str;
+use Illuminate\Validation\ValidationException;
+
+class LoginRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     */
```
Comment (kamilpiech97) [inline app/Http/Requests/Auth/LoginRequest.php]: Should be cleared

**Review by kamilpiech97 — CHANGES_REQUESTED** [review]

**Comment by kamilpiech97** [comment]
> Maybe we could start with Laravel 11?

Yep, I think that should be obligatory.

**Review by JakubKermes — APPROVED** [review]

**Review by krzysztofrewak — APPROVED** [review]

**Review by kamilpiech97 — APPROVED** [review]

**Pull request merged**